src/content/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx

```diff
@@ -1,38 +0,0 @@
----
```

Comment:
This page is currently being used for testing. Do you mind keeping this one around?